ironbank/hardening_manifest.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+
+# The repository name in registry1, excluding /ironbank/
+name: "coder/coder-enterprise/coder-service-2"
+
+# List of tags to push for the repository in registry1
+# The most specific version should be the first tag and will be shown
+# on ironbank.dso.mil
+tags:
+- "0.15.3"
+- "latest"
+
+# Build args passed to Dockerfile ARGs
+args:
+  BASE_IMAGE: "redhat/ubi/ubi8"
+  BASE_TAG: "8.7"
+  # Needs to be kept in sync with the resource below.
+  TERRAFORM_CODER_PROVIDER_VERSION: "0.6.9"
+
+# Docker image labels
+labels:
+  org.opencontainers.image.title: "coder-service-v2"
+  # Human-readable description of the software packaged in the image
+  org.opencontainers.image.description: "Coder server binary, includes REST API, Terraform, and dashboard"
+  # License(s) under which contained software is distributed
+  org.opencontainers.image.licenses: "APGL"
```

Comment:
AGPL* and also enterprise licensed, maybe we should just remove the licenses field?